README.md

```diff
@@ -157,6 +157,20 @@ sudo apt update
 sudo apt install xournalpp
 ```
 
+### Fedora
+
+The released version of xournalpp is available in the main repository via _Software_ or the following command:
+
+```bash
+sudo dnf install xournalpp
+```
+or 
+```bash
+pkcon install xournalpp
+```
+The bleeding edge packages synced to xournalpp git master on a daily basis are available from [COPR luya/xournalpp](https://copr.fedorainfracloud.org/coprs/luya/xournalpp/).
+
```

Comment:
Can you leave your contact information here since we (the Xournal++ team) are not the ones maintaining this package?

Reply:
Email address is luya_at_fedoraproject.org.

Reply:
Sorry, I mean in the text of the readme (since people who use the stable release will know who to contact).

Reply:
Requested change is done. The contact info is included in the link.